sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpPipelinePolicy.java

```diff
@@ -17,11 +18,22 @@
 @FunctionalInterface
 public interface HttpPipelinePolicy {
     /**
-     * Process provided request context and invokes the next policy.
+     * Processes provided request context and invokes the next policy.
      *
-     * @param context request context
+     * @param context The request context.
      * @param next The next policy to invoke.
      * @return A publisher that initiates the request upon subscription and emits a response on completion.
      */
     Mono<HttpResponse> process(HttpPipelineCallContext context, HttpPipelineNextPolicy next);
+
+    /**
+     * Gets the position to place the policy.
+     * <p>
+     * By default pipeline policies are positioned {@link HttpPipelinePosition#PER_RETRY}.
+     *
+     * @return The position to place the policy.
+     */
+    default HttpPipelinePosition getPipelinePosition() {
```

Comment:
I thought @JonathanGiles  suggested creating a separate interface `PositionalHttpPipelinePolicy extends HttpPipelinePolicy`

Reply:
Discussed offline, going with the default method on the interface as adding a sub-interface would require most `HttpPipelinePolicy` instance to be updated to implement the new interface.